node/tests/hubController.js

```diff
@@ -122,7 +125,11 @@ test.serial('translatePlatformsInvalidHmac', async t => {
 
     var verificationInfo = {};
     verificationInfo.key = config.subscriptionInfo.key;
-    verifcationInfo.hmac = "this_wont_match_the_hash";
+    verificationInfo.header = {
+        "X-Hub-Signature": "this_wont_match_the_hash"
+    };
 
-    t.throws(hubController.translatePlatforms(config.hubId, authInfo, config.subscription.sampleFeed, verificationInfo));
```

Comment:
so even with the removal of t.throws this is expected to fail? 

---
In reply to: [100192582](https://github.com/openT2T/cloud/pull/14#discussion_r100192582) [](ancestors = 100192582,100191664)